src/test/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+#DISABLE EUREKA
+eureka.client.enabled=false
+
+slack.slashCommandToken=slashCommandToken
+
+#Eureka client
+spring.application.name=keepers-slackbot
+server.port=0
```

Comment:
Could you help me
Why do we use zero value for server.port ?

Reply:
Default port is 8080 when server.port is removed.
If server.port=0 it means obtain port from Eureka

Reply:
ok, thanks I get it now